adapter/src/main/java/com/twilio/kudu/sql/KuduPrepareImpl.java

```diff
@@ -271,7 +271,7 @@ public void executeDdl(Context context, SqlNode node) {
               "GROUP BY clause should contain a FLOOR function on the timestamp column and an interval");
         }
 
-        String physicalCubeTableName = kuduTable.getName() + "-" + cubeName + "-" + interval + "-" + "Aggregation";
+        String physicalCubeTableName = kuduTable.getName() + "-" + cubeName + "_" + interval + "-" + "Aggregation";
```

Comment:
why do we need to change this? This would change the automated kick-off of kudu-alter-table jobs. https://code.hq.twilio.com/data/altruistic-armadillo/blob/master/src/enterprise_insight_dag/ArmadilloWorkflow.py#L157